runtime/js/40_testing.js

```diff
@@ -397,7 +397,11 @@
         const preResource = pre[resource];
         const postResource = post[resource];
         if (preResource === postResource) continue;
-        if (ArrayPrototypeIncludes(resourceAllowList, postResource)) continue;
+        if (
+          ArrayPrototypeIncludes(resourceAllowList, preResource || postResource)
```

Comment:
Is it possible that both `preResource` and `postResource` are non-zero / undefined? If so, then this `.includes(X || Y)` will only check for `X` and will ignore `Y`.

Reply:
Yes we want it to `Y` i.e `postResource` if `X` is truthy.

Reply:
Hmm, this was a bit hard to understand but I managed to explain it to myself well enough to believe that this is fine.

Reply:
yikes...i read my comment again. I meant "we want it to _ignore_ Y"

Reply:
Yeah, I managed to gather your meaning. It was the code that was throwing me on a loop :)